test/client/api.js

```diff
@@ -671,6 +671,18 @@ module.exports = config => {
     )
   }
 
+  ClientApi.prototype.consumeSigninCode = function (code, metricsContext) {
+    return this.doRequest(
+      'POST',
+      `${this.baseURL}/signinCodes/consume`,
+      null,
+      {
+        code: code.toString('base64').replace(/\+/g, '-').replace(/\//g, '_').replace(/=/g, ''),
```

Comment:
Is the conversion to a URL safe base64 number necessary, or just a defensive measure?